packages/backend/src/models/ContentModel/ContentConfigurations/DashboardContentConfiguration.ts

```diff
@@ -0,0 +1,148 @@
+import { ChartKind, ContentType, DashboardContent } from '@lightdash/common';
+import { Knex } from 'knex';
+import { DashboardsTableName } from '../../../database/entities/dashboards';
+import { OrganizationTableName } from '../../../database/entities/organizations';
+import { PinnedDashboardTableName } from '../../../database/entities/pinnedList';
+import { ProjectTableName } from '../../../database/entities/projects';
+import { SpaceTableName } from '../../../database/entities/spaces';
+import {
+    ContentConfiguration,
+    ContentFilters,
+    SummaryContentRow,
+} from '../ContentModelTypes';
+
+type SelectSavedSql = SummaryContentRow<{
+    source: 'sql';
+    chart_kind: ChartKind;
```

Comment:
confusing property for a dashboard

Reply:
That type is wrong. Going to update it 👍